### ISSUE TITLE
SA1412 code fix does not work properly on linked files

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1168]
When the SA1412 code fix is applied to a fix that is included in the project via a link, a copy of the original file is made at the target location of the link, and the new copy is included in the project instead of the original file.

**Comment by pdelvo** [comment]
I dont think this would work. I tried to make a text change to the document so that the encoding change is not the only change. but the encoding change was not working.

**Comment by sharwell** [comment]
Blocked on dotnet/roslyn#4767
